src/ui.data-table/UTable.vue

```diff
@@ -190,8 +190,8 @@
               :columns="columns"
               :config="config"
               :attrs="keysAttrs"
-              v-bind="getRowAttrs(row.id)"
-              :class="cx([getRowAttrs(row.id).class, row.class])"
+              v-bind="getRowAttrs(row)"
+              :class="cx([getRowAttrs(row).class, row.class])"
```

Comment:
here should be `:class="cx([getRowAttrs(row).class, getRowClasses(row)])"`

and `getRowClasses` should:

```js
function getRowClasses(row) {
  return typeof row?.class === "function" ? row.class(row) : row.class || "";
}
```

**Single responsibility principle!**